helm-charts/common/conversational-ui/values.yaml

```diff
@@ -0,0 +1,59 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Default values for conversational-ui.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: opea/chatqna-conversation-ui:latest
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  # tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
```

Comment:
Could we add securityContext back here? but set it as an empty dict, and comment out the supposed `right` values. Also add a comment line here saying it's related to a bug in conversational UI and leave the bug link in the comment

Reply:
Done